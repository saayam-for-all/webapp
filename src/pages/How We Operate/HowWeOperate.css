.blue-box {
  width: 500px; /* or any desired width */
  height: 300px; /* or make it square with same height */
  background-color: #eef3ff; /* light blue */
  border-radius: 24px; /* rounded corners */
  padding: 40px;
  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.05); /* optional soft shadow */
}
