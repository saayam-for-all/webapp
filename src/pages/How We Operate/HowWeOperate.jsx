import React from "react";
import "./HowWeOperate.css";
import howWeOperateData from "./data";

const HowWeOperate = () => {
  return (
    <div className="px-14 my-6">
      {howWeOperateData?.map((item, idx) => (
        <div key={idx}>
          <h1 className="text-2xl font-semibold mt-3">{item?.heading}</h1>
          {item?.points.map((point, index) => (
            <div key={index}>
              {!Array.isArray(point) ? (
                <ul className="list-disc text-lg pl-10">
                  <li>{point}</li>
                </ul>
              ) : (
                point.map((subPoint, subIndex) => (
                  <div key={subIndex}>
                    {!Array.isArray(subPoint) ? (
                      <ul
                        style={{
                          listStyleType: "circle",
                        }}
                        className="pl-20 text-lg"
                      >
                        <li>{subPoint}</li>
                      </ul>
                    ) : (
                      subPoint.map((subSubPoint, subSubIndex) => (
                        <ul
                          key={subSubIndex}
                          style={{
                            listStyleType: "square",
                          }}
                          className="pl-28 text-lg"
                        >
                          <li>{subSubPoint}</li>
                        </ul>
                      ))
                    )}
                  </div>
                ))
              )}
            </div>
          ))}
        </div>
      ))}
    </div>
  );
};

// recursive function to render nested list
// const HowWeOperate = () => {
// 	const renderList = (points, level) => {
// 		return (
// 			<ul
// 				className={`text-lg pl-8`}
// 				style={{
// 					listStyleType:
// 						level === 0
// 							? "disc"
// 							: level === 1
// 							? "circle"
// 							: "square",
// 				}}
// 			>
// 				{points.map((point, index) => (
// 					<li key={index}>
// 						{!Array.isArray(point)
// 							? point
// 							: renderList(point, level + 1)}
// 					</li>
// 				))}
// 			</ul>
// 		);
// 	};

// 	return (
// 		<div className='px-14 my-6'>
// 			{howWeOperateData?.map((item, idx) => (
// 				<div key={idx}>
// 					<h1 className='text-2xl font-semibold mt-3'>
// 						{item?.heading}
// 					</h1>
// 					{renderList(item.points, 0)}
// 				</div>
// 			))}
// 		</div>
// 	);
// };

export default HowWeOperate;
