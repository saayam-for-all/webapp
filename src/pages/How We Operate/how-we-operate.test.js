import { render } from "@testing-library/react";
import HowWeOperate from "./HowWeOperate";

describe("HowWeOperate", () => {
  it("renders correctly", () => {
    const tree = render(<HowWeOperate />);
    expect(tree).toMatchSnapshot();
  });
});
