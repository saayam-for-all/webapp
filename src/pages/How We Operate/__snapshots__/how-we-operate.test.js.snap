// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HowWeOperate renders correctly 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="font-sans"
      >
        <section
          class="text-center px-4 py-12 bg-white"
        >
          <h2
            class="text-3xl font-bold mb-4"
          >
            Volunteer Services
          </h2>
          <p
            class="text-gray-700 max-w-3xl mx-auto mb-4"
          >
            At Saayam for All, we connect individuals who need assistance with compassionate volunteers ready to help.
          </p>
          <p
            class="text-gray-700 max-w-3xl mx-auto mb-10"
          >
            Whether it's delivering groceries, providing companionship, or supporting small everyday tasks, our volunteers make a real difference—one request at a time. We believe in building a community where help is just a request away.
          </p>
          <div
            class="grid grid-cols-1 sm:grid-cols-3 gap-6 max-w-5xl mx-auto"
          >
            <img
              alt="Grocery Delivery"
              class="rounded-xl object-cover h-64 w-full"
              src="s5"
            />
            <img
              alt="Group Volunteering"
              class="rounded-xl object-cover h-64 w-full"
              src="s5"
            />
            <img
              alt="Beach Cleanup"
              class="rounded-xl object-cover h-64 w-full"
              src="s5"
            />
          </div>
        </section>
        <div
          class="py-16 px-4"
        >
          <div
            class="max-w-4xl mx-auto"
          >
            <h2
              class="text-3xl font-bold text-center mb-4"
            >
              How We Operate
            </h2>
            <p
              class="text-center text-gray-600 mb-12"
            >
              At Saayam For All, getting help is as simple and seamless as requesting a ride.
              <br />
              Get help in 5 easy steps.
            </p>
            <div
              class="relative h-full min-h-[800px]"
            >
              <div
                class="absolute left-1/2 top-28 transform -translate-x-1/2 w-0.5 border-l-2 border-dashed border-blue-300 z-10"
              />
              <div
                class="step-box"
              >
                <div
                  class="relative flex flex-col sm:flex-row sm:flex-row-reverse items-center mb-16"
                >
                  <div
                    class="absolute top-1/2 left-1/2 -translate-x-1/2 -translate-y-1/2 w-11/12 sm:w-3/4 h-72 sm:h-64 bg-blue-50 rounded-2xl z-0"
                  />
                  <div
                    class="w-full sm:w-1/2 flex justify-center z-10 mb-6 sm:mb-0 order-1 sm:order-none"
                  >
                    <img
                      alt="Request for Help"
                      class="rounded-xl object-cover h-40 sm:h-60 lg:h-60"
                      src="s5"
                    />
                  </div>
                  <div
                    class="z-20 mb-4 sm:mb-0 absolute left-1/2 top-[88px] sm:static sm:left-auto sm:top-auto sm:transform-none -translate-x-1/2"
                  >
                    <div
                      class="w-12 h-12 sm:w-14 sm:h-14 rounded-full bg-blue-500 text-white flex items-center justify-center font-bold text-lg sm:text-xl shadow-lg"
                    >
                      1
                    </div>
                  </div>
                  <div
                    class="w-full sm:w-1/2 z-10 flex order-2 sm:order-none sm:pl-16 sm:justify-start sm:-translate-x-12"
                  >
                    <div
                      class="my-auto text-center sm:text-left px-4 sm:px-0"
                    >
                      <h3
                        class="text-xl font-bold leading-tight mb-2"
                      >
                        Request for Help
                      </h3>
                      <p
                        class="text-gray-600"
                      >
                        Submit a request for the support you need—just like booking a ride through an app.
                      </p>
                    </div>
                  </div>
                </div>
                <div
                  class="relative flex flex-col sm:flex-row  items-center mb-16"
                >
                  <div
                    class="absolute top-1/2 left-1/2 -translate-x-1/2 -translate-y-1/2 w-11/12 sm:w-3/4 h-72 sm:h-64 bg-blue-50 rounded-2xl z-0"
                  />
                  <div
                    class="w-full sm:w-1/2 flex justify-center z-10 mb-6 sm:mb-0 order-1 sm:order-none"
                  >
                    <img
                      alt=" Get Matching With a Volunteer"
                      class="rounded-xl object-cover h-40 sm:h-60 lg:h-60"
                      src="s5"
                    />
                  </div>
                  <div
                    class="z-20 mb-4 sm:mb-0 absolute left-1/2 top-[88px] sm:static sm:left-auto sm:top-auto sm:transform-none -translate-x-1/2"
                  >
                    <div
                      class="w-12 h-12 sm:w-14 sm:h-14 rounded-full bg-blue-500 text-white flex items-center justify-center font-bold text-lg sm:text-xl shadow-lg"
                    >
                      2
                    </div>
                  </div>
                  <div
                    class="w-full sm:w-1/2 z-10 flex order-2 sm:order-none sm:pr-16 sm:justify-end sm:translate-x-12"
                  >
                    <div
                      class="my-auto text-center sm:text-left px-4 sm:px-0"
                    >
                      <h3
                        class="text-xl font-bold leading-tight mb-2"
                      >
                         Get Matching With a Volunteer
                      </h3>
                      <p
                        class="text-gray-600"
                      >
                        We don't just match—you get the smartest match possible.Our AI/ML engine find the right volunteer for you faster and more accuately, just like a ride share app- but smarter, more human, built for community care .
                      </p>
                    </div>
                  </div>
                </div>
                <div
                  class="relative flex flex-col sm:flex-row sm:flex-row-reverse items-center mb-16"
                >
                  <div
                    class="absolute top-1/2 left-1/2 -translate-x-1/2 -translate-y-1/2 w-11/12 sm:w-3/4 h-72 sm:h-64 bg-blue-50 rounded-2xl z-0"
                  />
                  <div
                    class="w-full sm:w-1/2 flex justify-center z-10 mb-6 sm:mb-0 order-1 sm:order-none"
                  >
                    <img
                      alt="Confirm Your Volunteer"
                      class="rounded-xl object-cover h-40 sm:h-60 lg:h-60"
                      src="s5"
                    />
                  </div>
                  <div
                    class="z-20 mb-4 sm:mb-0 absolute left-1/2 top-[88px] sm:static sm:left-auto sm:top-auto sm:transform-none -translate-x-1/2"
                  >
                    <div
                      class="w-12 h-12 sm:w-14 sm:h-14 rounded-full bg-blue-500 text-white flex items-center justify-center font-bold text-lg sm:text-xl shadow-lg"
                    >
                      3
                    </div>
                  </div>
                  <div
                    class="w-full sm:w-1/2 z-10 flex order-2 sm:order-none sm:pl-16 sm:justify-start sm:-translate-x-12"
                  >
                    <div
                      class="my-auto text-center sm:text-left px-4 sm:px-0"
                    >
                      <h3
                        class="text-xl font-bold leading-tight mb-2"
                      >
                        Confirm Your Volunteer
                      </h3>
                      <p
                        class="text-gray-600"
                      >
                        From these potential matches, one volunteer will be chosen to assist you, much like selecting a specific car.
                      </p>
                    </div>
                  </div>
                </div>
                <div
                  class="relative flex flex-col sm:flex-row  items-center mb-16"
                >
                  <div
                    class="absolute top-1/2 left-1/2 -translate-x-1/2 -translate-y-1/2 w-11/12 sm:w-3/4 h-72 sm:h-64 bg-blue-50 rounded-2xl z-0"
                  />
                  <div
                    class="w-full sm:w-1/2 flex justify-center z-10 mb-6 sm:mb-0 order-1 sm:order-none"
                  >
                    <img
                      alt="Connect and Coordinate"
                      class="rounded-xl object-cover h-40 sm:h-60 lg:h-60"
                      src="s5"
                    />
                  </div>
                  <div
                    class="z-20 mb-4 sm:mb-0 absolute left-1/2 top-[88px] sm:static sm:left-auto sm:top-auto sm:transform-none -translate-x-1/2"
                  >
                    <div
                      class="w-12 h-12 sm:w-14 sm:h-14 rounded-full bg-blue-500 text-white flex items-center justify-center font-bold text-lg sm:text-xl shadow-lg"
                    >
                      4
                    </div>
                  </div>
                  <div
                    class="w-full sm:w-1/2 z-10 flex order-2 sm:order-none sm:pr-16 sm:justify-end sm:translate-x-12"
                  >
                    <div
                      class="my-auto text-center sm:text-left px-4 sm:px-0"
                    >
                      <h3
                        class="text-xl font-bold leading-tight mb-2"
                      >
                        Connect and Coordinate
                      </h3>
                      <p
                        class="text-gray-600"
                      >
                        Your volunteer reaches out to finalize details and schedule the support you need.
                      </p>
                    </div>
                  </div>
                </div>
                <div
                  class="relative flex flex-col sm:flex-row sm:flex-row-reverse items-center mb-16"
                >
                  <div
                    class="absolute top-1/2 left-1/2 -translate-x-1/2 -translate-y-1/2 w-11/12 sm:w-3/4 h-72 sm:h-64 bg-blue-50 rounded-2xl z-0"
                  />
                  <div
                    class="w-full sm:w-1/2 flex justify-center z-10 mb-6 sm:mb-0 order-1 sm:order-none"
                  >
                    <img
                      alt="Receive Assistance"
                      class="rounded-xl object-cover h-40 sm:h-60 lg:h-60"
                      src="s5"
                    />
                  </div>
                  <div
                    class="z-20 mb-4 sm:mb-0 absolute left-1/2 top-[88px] sm:static sm:left-auto sm:top-auto sm:transform-none -translate-x-1/2"
                  >
                    <div
                      class="w-12 h-12 sm:w-14 sm:h-14 rounded-full bg-blue-500 text-white flex items-center justify-center font-bold text-lg sm:text-xl shadow-lg"
                    >
                      5
                    </div>
                  </div>
                  <div
                    class="w-full sm:w-1/2 z-10 flex order-2 sm:order-none sm:pl-16 sm:justify-start sm:-translate-x-12"
                  >
                    <div
                      class="my-auto text-center sm:text-left px-4 sm:px-0"
                    >
                      <h3
                        class="text-xl font-bold leading-tight mb-2"
                      >
                        Receive Assistance
                      </h3>
                      <p
                        class="text-gray-600"
                      >
                        The volunteer provides the help you requested,working with other if needed-making sure you reach your destination.
                      </p>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="text-center mt-32"
            >
              <span
                class="text-blue-500 text-2xl font-bold"
              >
                Connecting help to those who need it.
              </span>
            </div>
            <div
              class="text-center mt-32"
            >
              <h3
                class="text-2xl font-bold mb-4"
              >
                Looking to volunteer with us?
              </h3>
              <p
                class="text-gray-600 mb-6"
              >
                Chat with our community and get in touch with us!
              </p>
              <button
                class="bg-blue-500 text-white px-6 py-3 rounded-lg hover:bg-blue-600 transition-colors"
              >
                Join the community
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="font-sans"
    >
      <section
        class="text-center px-4 py-12 bg-white"
      >
        <h2
          class="text-3xl font-bold mb-4"
        >
          Volunteer Services
        </h2>
        <p
          class="text-gray-700 max-w-3xl mx-auto mb-4"
        >
          At Saayam for All, we connect individuals who need assistance with compassionate volunteers ready to help.
        </p>
        <p
          class="text-gray-700 max-w-3xl mx-auto mb-10"
        >
          Whether it's delivering groceries, providing companionship, or supporting small everyday tasks, our volunteers make a real difference—one request at a time. We believe in building a community where help is just a request away.
        </p>
        <div
          class="grid grid-cols-1 sm:grid-cols-3 gap-6 max-w-5xl mx-auto"
        >
          <img
            alt="Grocery Delivery"
            class="rounded-xl object-cover h-64 w-full"
            src="s5"
          />
          <img
            alt="Group Volunteering"
            class="rounded-xl object-cover h-64 w-full"
            src="s5"
          />
          <img
            alt="Beach Cleanup"
            class="rounded-xl object-cover h-64 w-full"
            src="s5"
          />
        </div>
      </section>
      <div
        class="py-16 px-4"
      >
        <div
          class="max-w-4xl mx-auto"
        >
          <h2
            class="text-3xl font-bold text-center mb-4"
          >
            How We Operate
          </h2>
          <p
            class="text-center text-gray-600 mb-12"
          >
            At Saayam For All, getting help is as simple and seamless as requesting a ride.
            <br />
            Get help in 5 easy steps.
          </p>
          <div
            class="relative h-full min-h-[800px]"
          >
            <div
              class="absolute left-1/2 top-28 transform -translate-x-1/2 w-0.5 border-l-2 border-dashed border-blue-300 z-10"
            />
            <div
              class="step-box"
            >
              <div
                class="relative flex flex-col sm:flex-row sm:flex-row-reverse items-center mb-16"
              >
                <div
                  class="absolute top-1/2 left-1/2 -translate-x-1/2 -translate-y-1/2 w-11/12 sm:w-3/4 h-72 sm:h-64 bg-blue-50 rounded-2xl z-0"
                />
                <div
                  class="w-full sm:w-1/2 flex justify-center z-10 mb-6 sm:mb-0 order-1 sm:order-none"
                >
                  <img
                    alt="Request for Help"
                    class="rounded-xl object-cover h-40 sm:h-60 lg:h-60"
                    src="s5"
                  />
                </div>
                <div
                  class="z-20 mb-4 sm:mb-0 absolute left-1/2 top-[88px] sm:static sm:left-auto sm:top-auto sm:transform-none -translate-x-1/2"
                >
                  <div
                    class="w-12 h-12 sm:w-14 sm:h-14 rounded-full bg-blue-500 text-white flex items-center justify-center font-bold text-lg sm:text-xl shadow-lg"
                  >
                    1
                  </div>
                </div>
                <div
                  class="w-full sm:w-1/2 z-10 flex order-2 sm:order-none sm:pl-16 sm:justify-start sm:-translate-x-12"
                >
                  <div
                    class="my-auto text-center sm:text-left px-4 sm:px-0"
                  >
                    <h3
                      class="text-xl font-bold leading-tight mb-2"
                    >
                      Request for Help
                    </h3>
                    <p
                      class="text-gray-600"
                    >
                      Submit a request for the support you need—just like booking a ride through an app.
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="relative flex flex-col sm:flex-row  items-center mb-16"
              >
                <div
                  class="absolute top-1/2 left-1/2 -translate-x-1/2 -translate-y-1/2 w-11/12 sm:w-3/4 h-72 sm:h-64 bg-blue-50 rounded-2xl z-0"
                />
                <div
                  class="w-full sm:w-1/2 flex justify-center z-10 mb-6 sm:mb-0 order-1 sm:order-none"
                >
                  <img
                    alt=" Get Matching With a Volunteer"
                    class="rounded-xl object-cover h-40 sm:h-60 lg:h-60"
                    src="s5"
                  />
                </div>
                <div
                  class="z-20 mb-4 sm:mb-0 absolute left-1/2 top-[88px] sm:static sm:left-auto sm:top-auto sm:transform-none -translate-x-1/2"
                >
                  <div
                    class="w-12 h-12 sm:w-14 sm:h-14 rounded-full bg-blue-500 text-white flex items-center justify-center font-bold text-lg sm:text-xl shadow-lg"
                  >
                    2
                  </div>
                </div>
                <div
                  class="w-full sm:w-1/2 z-10 flex order-2 sm:order-none sm:pr-16 sm:justify-end sm:translate-x-12"
                >
                  <div
                    class="my-auto text-center sm:text-left px-4 sm:px-0"
                  >
                    <h3
                      class="text-xl font-bold leading-tight mb-2"
                    >
                       Get Matching With a Volunteer
                    </h3>
                    <p
                      class="text-gray-600"
                    >
                      We don't just match—you get the smartest match possible.Our AI/ML engine find the right volunteer for you faster and more accuately, just like a ride share app- but smarter, more human, built for community care .
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="relative flex flex-col sm:flex-row sm:flex-row-reverse items-center mb-16"
              >
                <div
                  class="absolute top-1/2 left-1/2 -translate-x-1/2 -translate-y-1/2 w-11/12 sm:w-3/4 h-72 sm:h-64 bg-blue-50 rounded-2xl z-0"
                />
                <div
                  class="w-full sm:w-1/2 flex justify-center z-10 mb-6 sm:mb-0 order-1 sm:order-none"
                >
                  <img
                    alt="Confirm Your Volunteer"
                    class="rounded-xl object-cover h-40 sm:h-60 lg:h-60"
                    src="s5"
                  />
                </div>
                <div
                  class="z-20 mb-4 sm:mb-0 absolute left-1/2 top-[88px] sm:static sm:left-auto sm:top-auto sm:transform-none -translate-x-1/2"
                >
                  <div
                    class="w-12 h-12 sm:w-14 sm:h-14 rounded-full bg-blue-500 text-white flex items-center justify-center font-bold text-lg sm:text-xl shadow-lg"
                  >
                    3
                  </div>
                </div>
                <div
                  class="w-full sm:w-1/2 z-10 flex order-2 sm:order-none sm:pl-16 sm:justify-start sm:-translate-x-12"
                >
                  <div
                    class="my-auto text-center sm:text-left px-4 sm:px-0"
                  >
                    <h3
                      class="text-xl font-bold leading-tight mb-2"
                    >
                      Confirm Your Volunteer
                    </h3>
                    <p
                      class="text-gray-600"
                    >
                      From these potential matches, one volunteer will be chosen to assist you, much like selecting a specific car.
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="relative flex flex-col sm:flex-row  items-center mb-16"
              >
                <div
                  class="absolute top-1/2 left-1/2 -translate-x-1/2 -translate-y-1/2 w-11/12 sm:w-3/4 h-72 sm:h-64 bg-blue-50 rounded-2xl z-0"
                />
                <div
                  class="w-full sm:w-1/2 flex justify-center z-10 mb-6 sm:mb-0 order-1 sm:order-none"
                >
                  <img
                    alt="Connect and Coordinate"
                    class="rounded-xl object-cover h-40 sm:h-60 lg:h-60"
                    src="s5"
                  />
                </div>
                <div
                  class="z-20 mb-4 sm:mb-0 absolute left-1/2 top-[88px] sm:static sm:left-auto sm:top-auto sm:transform-none -translate-x-1/2"
                >
                  <div
                    class="w-12 h-12 sm:w-14 sm:h-14 rounded-full bg-blue-500 text-white flex items-center justify-center font-bold text-lg sm:text-xl shadow-lg"
                  >
                    4
                  </div>
                </div>
                <div
                  class="w-full sm:w-1/2 z-10 flex order-2 sm:order-none sm:pr-16 sm:justify-end sm:translate-x-12"
                >
                  <div
                    class="my-auto text-center sm:text-left px-4 sm:px-0"
                  >
                    <h3
                      class="text-xl font-bold leading-tight mb-2"
                    >
                      Connect and Coordinate
                    </h3>
                    <p
                      class="text-gray-600"
                    >
                      Your volunteer reaches out to finalize details and schedule the support you need.
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="relative flex flex-col sm:flex-row sm:flex-row-reverse items-center mb-16"
              >
                <div
                  class="absolute top-1/2 left-1/2 -translate-x-1/2 -translate-y-1/2 w-11/12 sm:w-3/4 h-72 sm:h-64 bg-blue-50 rounded-2xl z-0"
                />
                <div
                  class="w-full sm:w-1/2 flex justify-center z-10 mb-6 sm:mb-0 order-1 sm:order-none"
                >
                  <img
                    alt="Receive Assistance"
                    class="rounded-xl object-cover h-40 sm:h-60 lg:h-60"
                    src="s5"
                  />
                </div>
                <div
                  class="z-20 mb-4 sm:mb-0 absolute left-1/2 top-[88px] sm:static sm:left-auto sm:top-auto sm:transform-none -translate-x-1/2"
                >
                  <div
                    class="w-12 h-12 sm:w-14 sm:h-14 rounded-full bg-blue-500 text-white flex items-center justify-center font-bold text-lg sm:text-xl shadow-lg"
                  >
                    5
                  </div>
                </div>
                <div
                  class="w-full sm:w-1/2 z-10 flex order-2 sm:order-none sm:pl-16 sm:justify-start sm:-translate-x-12"
                >
                  <div
                    class="my-auto text-center sm:text-left px-4 sm:px-0"
                  >
                    <h3
                      class="text-xl font-bold leading-tight mb-2"
                    >
                      Receive Assistance
                    </h3>
                    <p
                      class="text-gray-600"
                    >
                      The volunteer provides the help you requested,working with other if needed-making sure you reach your destination.
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="text-center mt-32"
          >
            <span
              class="text-blue-500 text-2xl font-bold"
            >
              Connecting help to those who need it.
            </span>
          </div>
          <div
            class="text-center mt-32"
          >
            <h3
              class="text-2xl font-bold mb-4"
            >
              Looking to volunteer with us?
            </h3>
            <p
              class="text-gray-600 mb-6"
            >
              Chat with our community and get in touch with us!
            </p>
            <button
              class="bg-blue-500 text-white px-6 py-3 rounded-lg hover:bg-blue-600 transition-colors"
            >
              Join the community
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
